platform/security/pdp/security-pdp-authzrealm/src/main/java/ddf/security/pdp/realm/xacml/XACMLConstants.java

```diff
@@ -22,6 +22,10 @@
 
     public static final String BOOLEAN_DATA_TYPE = "http://www.w3.org/2001/XMLSchema#boolean";
 
+    public static final String INTEGER_DATA_TYPE = "http://www.w3.org/2001/XMLSchema#integer";
```

Comment:
This was already here, but we should removing XACMLConstants (okay with it being another ticket) and moving the constants to XacmlPdp. The constants are only being used by XacmlPdp anyway.